lib/import-export-visitor.js

```diff
@@ -303,15 +318,16 @@ IEVp.finalizeHoisting = function () {
     delete bodyInfo.hoistedExportsMap;
     delete bodyInfo.hoistedExportsString;
     delete bodyInfo.hoistedImportsString;
-  }, this);
+  }
 
   // Just in case we call finalizeHoisting again, don't hoist anything.
   this.bodyInfos.length = 0;
 
-  this.removals.forEach(processRemoval);
+  let removalCount = this.removals.length;
 
-  // Just in case we call finalizeHoisting again, don't remove anything.
-  this.removals.length = 0;
+  while (removalCount--) {
+    processRemoval(this.removals.pop());
```

Comment:
btw I didn't see your reply about once per file. I ok with omitting it now to get the greater win of more ES6 but it'll be something I think I'll run into again with, know wait a minute is built-in use good here? (as I forget the context of the call).